KeywordSearch/src/org/sleuthkit/autopsy/keywordsearch/DropdownListSearchPanel.java

```diff
@@ -200,7 +200,7 @@ private void initComponents() {
 
         setFont(getFont().deriveFont(getFont().getStyle() & ~java.awt.Font.BOLD, 11));
 
-        jSplitPane1.setFont(jSplitPane1.getFont().deriveFont(jSplitPane1.getFont().getStyle() & ~java.awt.Font.BOLD, 11));
+        jSplitPane1.setFont(leftPane.getFont());
 
```

Comment:
this looks wrong to me.  the font for leftpane is not assigned until the next couple of lines